lib/tool_belt/repository.rb

```diff
@@ -20,13 +20,17 @@ def self.parse(config)
       end
     end
 
+    def tag_exists(tag)
```

Comment:
nitpick: methods that return true/false should end with a ?. `def tag_exists?(tag)`